### PR TITLE
Update configure-biztalk-server.md

### PR DESCRIPTION
**Pull request by SanjivGupta-SanGupt**

### CODE REVIEW

**Comment by SanjivGupta-SanGupt** [comment]
Made few cosmetic changes

File: biztalk/install-and-config-guides/configure-biztalk-server.md [hunk 1]
```diff
@@ -249,37 +249,37 @@ BAM alerts require BAM tools to be enabled.
...
+BizTalk TMS is a service that refreshes authentication tokens used by BizTalk, it is a prerequisite for some adapters, i.e. Office 365 series adapters.  
 
 >[!IMPORTANT]
 ><li>TMS must be configured on a server which also has the "Enterprise Single Sign-On Service" (ENTSSO) running.</li>
-><li>TMS should only be configured on one server in a BizTalk Group. For high-availability scenarios, you may configure TMS on multiple servers in an Active-Passive topology.</li>
+><li>TMS should only be configured on a single server in a BizTalk Group. For high-availability scenarios, you may configure TMS on multiple servers in an Active-Passive topology.</li>
 
 >[!NOTE]
->BizTalk TMS configuration is available start from **BizTalk Server 2020**.
+>Applicable starting **BizTalk Server 2020**.
```
Comment (msnehagup) [inline biztalk/install-and-config-guides/configure-biztalk-server.md]: Shouldn't this note come at the beginning of this section, right below the heading?

File: biztalk/install-and-config-guides/configure-biztalk-server.md [hunk 2]
```diff
@@ -249,37 +249,37 @@ BAM alerts require BAM tools to be enabled.
...
+    |Windows groups | BizTalk REST APIs use Windows authentication, specify the Windows groups allowed to access the APIs here. <ul><li>Management REST APIs User Role: Specify name of Windows group as an extra layer of security. Once specified, the requester access to BizTalk information at REST endpoint is limited to what is configured in BizTalk for the specified Windows group(s).</li><li>Operational Data REST APIs User Role: Specify Windows group name that you wish to use to control the access.</li>|
+    |REST APIs website|Select the Web site to host both the REST APIs. |
 
 ### Configure BizTalk TMS
 
-BizTalk TMS is a service that refreshes the authentication tokens used by BizTalk, it is a prerequisite for some adapters, i.e. Office 365 series adapters.
```
Comment (msnehagup) [inline biztalk/install-and-config-guides/configure-biztalk-server.md:267]: ... tokens used by BizTalk. It is a prerequisite for the Office 365 adapters.

File: biztalk/install-and-config-guides/configure-biztalk-server.md [hunk 3]
```diff
@@ -249,37 +249,37 @@ BAM alerts require BAM tools to be enabled.
...
+    |Web service accounts | Specify the account used to run the IIS services. <ul><li>Management Service: REST APIs for BizTalk Server artifacts management. The default URI for **Management Service** APIs is http(s)://localhost/BizTalkManagementService. </li><li>Operational Data Service: Service providing information on the instances and messages flowing through your BizTalk Server environment. The operational data feed is the same data you get looking at Group Hub in BizTalk Server Administration. The default URI for **Operational Data service** APIs is http(s)://localhost/BizTalkOperationalDataService.</li></ul>|
+    |Windows groups | BizTalk REST APIs use Windows authentication, specify the Windows groups allowed to access the APIs here. <ul><li>Management REST APIs User Role: Specify name of Windows group as an extra layer of security. Once specified, the requester access to BizTalk information at REST endpoint is limited to what is configured in BizTalk for the specified Windows group(s).</li><li>Operational Data REST APIs User Role: Specify Windows group name that you wish to use to control the access.</li>|
```
Comment (msnehagup) [inline biztalk/install-and-config-guides/configure-biztalk-server.md]: I still find this text somewhat confusing. Maybe @pariksg / @pravagar  have some suggestion here?

Comment (msnehagup) [inline biztalk/install-and-config-guides/configure-biztalk-server.md:262]: I still find the text for the Mgmt API somewhat confusing. Maybe @pariksg / @pravagar  have some suggestion here?

**Review by msnehagup — APPROVED** [review]

File: biztalk/install-and-config-guides/configure-biztalk-server.md (hunk 1 above)
Comment (SanjivGupta-SanGupt) [reply biztalk/install-and-config-guides/configure-biztalk-server.md]: Agree

File: biztalk/install-and-config-guides/configure-biztalk-server.md (hunk 2 above)
Comment (SanjivGupta-SanGupt) [reply biztalk/install-and-config-guides/configure-biztalk-server.md:267]: 2020 specific. Who knows what is the future of TMS. But we can change it as you say to be more accurate for 2020.

**Pull request merged**